src/data/moves/move-attrs/ability-copy-attr.ts

```diff
@@ -64,4 +66,27 @@ export class AbilityCopyAttr extends MoveEffectAttr {
       return ret;
     };
   }
+
+  /**
+   * If the target's ability is detrimental, grants a (-5) penalty.
+   * Otherwise, grants (+2) effect score if one Pokemon's detrimental ability is overridden
+   * by this effect, and (+3) if more than one Pokemon's detrimental ability is overridden.
+   * @see {@linkcode DETRIMENTAL_ABILITIES}
+   */
+  override getEffectScore(user: EnemyPokemon, target: Pokemon, _move: Move): number {
+    const targetRevealedAbilityId = target.getAbilities({ revealedOnly: true }).find((ab) => !ab.passive)?.ability.id;
+
+    if (!targetRevealedAbilityId || DETRIMENTAL_ABILITIES.includes(targetRevealedAbilityId)) {
+      return -5;
```

Comment:
This `-5` number should probably be a descriptive constant value

and not a _magic number_